bundles/core/org.eclipse.smarthome.core.persistence/src/main/java/org/eclipse/smarthome/core/persistence/rest/ItemHistoryBean.java

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright (c) 2010-2013, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.core.persistence.rest;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.eclipse.smarthome.core.library.types.DecimalType;
+import org.eclipse.smarthome.core.library.types.OnOffType;
+import org.eclipse.smarthome.core.library.types.OpenClosedType;
+import org.eclipse.smarthome.core.types.State;
+
+/**
+ * This is a java bean that is used with JAXB to serialize items to JSON.
+ *
+ * @author Chris Jackson - Initial Contribution
+ *
+ */
+public class ItemHistoryBean {
+
+    public String name;
+    // public String type;
+    public String totalrecords;
+    // public String link;
+
+    // public String icon;
+    // public String label;
+    // public String units;
+    // public String format;
+    // public String map;
+
+    // public List<String> groups;
+    // public List<String> services;
+
+    public String timestart;
+    public String timeend;
+    public String statemax;
+    public String timemax;
+    public String statemin;
+    public String timemin;
+    public String stateavg;
+    public String datapoints;
+
+    public List<HistoryDataBean> data;
+
+    public ItemHistoryBean() {
+    };
+
+    public double addData(Long time, State state) {
```

Comment:
Could you add JavaDoc, at least the return value needs an explanation.
For example: a numeric value that could represent the state. It could be used for e.g. min / max / avg calculation.